Alien vault search alerts bug fix (#14371)
demisto/content: Packs/AlienVault_USM_Anywhere/ReleaseNotes/1_0_5.md

#### Integrations
##### AlienVault USM Anywhere

> - Updated the Docker image to: *demisto/python3:3.9.5.21272*.

Reviewer: ```suggestion
- Updated the Docker image to: *demisto/python3:3.9.6.22912*.
```

> - Fixed bug where alarm data raised an exception when it was parsed.

Reviewer: ```suggestion
- Fixed an issue where the integration failed to parse dates from the alarm data.
```
